tests/travis/checkout_test_against_branch.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SCRIPT_DIR=`dirname $0`
+
+if [ "$TEST_AGAINST_PIWIK_BRANCH" == "" ]; then
+    if [ "$TEST_AGAINST_CORE" == "latest_stable" ]; then # test against the latest stable release of piwik core (including betas & release candidates)
```

Comment:
I don't really want to rename it since it will break existing builds that may use it... I guess it's ok if we're the only ones using though.

And no, it might not be used. It's main use is to see if there's a change in piwik core's master that's causing the build failure, rather than the plugin (ie, if the plugin build fails w/ master, but the build w/ lastest_stable succeeds, maybe the issue isn't w/ the plugin). Was requested by pro devs around meetup time.